lib/task.js

```diff
@@ -224,8 +224,8 @@ export default class Task {
     };
 
     // Zero is a valid option so only check for existence.
-    if ('cpuset' in this.options) {
-      procConfig.create.Cpuset = this.options.cpuset;
+    if ('cpusetCpus' in this.options) {
+      procConfig.create.HostConfig.CpusetCpus = this.options.cpusetCpus;
```

Comment:
Yea they are deprecating the other option so figured now is a good time to switch